infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationDiscoveryService.java

```diff
@@ -708,7 +733,34 @@ public void updateTopology(LogReplicationClusterInfo.TopologyConfigurationMsg to
         return null;
     }
 
+    @Override
+    public void forceSnapshotSync(String clusterId) {
```

Comment:
should we have a forceSnapshotSync API that forces snapshot sync to all standbys?

Reply:
This is no requirement for this.